client/components/project.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import Button from './button';
+import Editor from './editor';
+
+export default class Project extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      language: 'HTML',
+      html: '',
+      css: '',
+      js: '',
+      // eslint-disable-next-line
+      srcDoc: ``
```

Comment:
You can make this an empty string like the rest and remove your eslint directive